share-credential-modal.html

```diff
@@ -0,0 +1,10 @@
+<br-modal br-title="Share Credential">
+  <div name="br-modal-body">
+    <input type="text" value="{{shareUrl}}" br-select-text="true" size="50"></input><br>
+    Press <strong>Ctrl + C</strong> to copy URL
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd
Also Ctrl-C is not universal.  For instance on Macs.